.github/workflows/pullpreview.yml

```diff
@@ -96,6 +106,11 @@ jobs:
           instance_type: ${{ inputs.INSTANCE_TYPE }}
           # Ports to open on the server
           ports: 80, 443
+          # It seems, although not identical, <condition> && <true-value> || <false-value> works in most cases.
```

Comment:
Identical to what? What do you mean with most cases?

Reply:
This comment wasn't supposed to be there. Let's just pretend you didn't see anything. Removed the Line...